src/Extensions/src/App.Metrics.Extensions.HealthChecks/App.Metrics.Extensions.HealthChecks.csproj

```diff
@@ -8,7 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="App.Metrics.Core" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
```

Comment:
Why no version?

Reply:
Please check `Directory.Build.targets` in the top-level directory. Versions are globally updated there.

Reply:
Great thnx for clarification